lib/peek/rblineprof/syntax_highlighter.rb

```diff
@@ -0,0 +1,38 @@
+require 'peek/rblineprof/highlighters/pygments_highlighter'
+require 'peek/rblineprof/highlighters/rouge_highlighter'
+
+module Peek
+  module Rblineprof
+    class SyntaxHighlighter
+      class_attribute :highlighter
```

Comment:
Could we make `SyntaxHighlighter.highlighter` lazy so that we can cleanly set it to a custom highlighter in an initializer? We have this need for GitLab where we're using a custom `Gitlab::Highlight` class.